app/routes/questions/$question.tsx

```diff
@@ -219,12 +219,14 @@ function Contents({html, glossary}: {html: string; glossary: Glossary}) {
 
     el.querySelectorAll('.glossary-entry').forEach((e) => {
       const entry = e.textContent && glossary[e?.textContent.toLowerCase().trim()]
-      // It's possible for a glossary entry to contain another one (e.g. 'goodness' and 'good'), so
-      // if this entry is a subset of a bigger entry, remove it. Ditto for entries that are parts of
-      // external links, as that could get confusing
       if (
+        // It's possible for a glossary entry to contain another one (e.g. 'goodness' and 'good'), so
+        // if this entry is a subset of a bigger entry, remove it.
         e.parentElement?.classList.contains('glossary-entry') ||
+        // Remove entries that are parts of external links, as that could get confusing
         e.parentElement?.tagName == 'A' ||
+        // Remove entries that point to the current question
```

Comment:
recursive glossary 😅

Reply:
yup :D

Reply:
this actually is fully recursive, it turns out. Which would be fun if all links are enabled... :D